daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1467,6 +1541,7 @@ message Module {
   repeated DefDataType data_types = 5;
   repeated DefValue values = 6;
   repeated DefTemplate templates = 7;
+  repeated DefException exceptions = 10; // *Available in versions >= 1.dev* (#8020)
```

Comment:
The numbering in this message is wild. Wow!

Reply:
standard protobuf